### PR TITLE
Added missing descriptions for attributes.

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:

- [ ] Content update

## Summary

When this pull request is merged, it will...
Add missing descriptions for attributes.

### CODE REVIEW

**Comment by dobooth** [comment]
Li,
 There were 4 attributes I couldn't figure out. I would love suggestions for completing them.
dontSeeElementInDOM
    parameterArray:
    attribute Array:
 
formatMoney AND mSetLocale
    userInput:
    locale:
I assume locale is a PHP locale value. Not sure what userInput would be…
 
getData
    index:
 
scrollTo
   selectorArray:

File: mftf/2.3/test/actions.md [hunk 1]
```diff
@@ -2458,8 +2463,8 @@ See [waitForJS docs on codeception.com](http://codeception.com/docs/modules/WebD
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`function`|string|optional|
-`time`|string|optional|
+`function`|string|optional| The function to be run after all javascript finishes.
```
Comment (jilu1) [inline mftf/2.3/test/actions.md]: Not accurate. It actually executes the JavaScript and waits up to $timeout seconds for it to return true.

File: mftf/2.3/test/actions.md [hunk 2]
```diff
@@ -2377,9 +2382,9 @@ See [waitForElementChange docs on codeception.com](http://codeception.com/docs/m
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`function`|string|optional|
-`time`|string|optional|
+`selector`|string|optional| The selector identifying the HTML element to be changed.
+`function`|string|optional| The function to be run after the element changes.
```
Comment (jilu1) [inline mftf/2.3/test/actions.md]: This is the callback function that will determine the element is changed.

File: mftf/2.3/test/actions.md [hunk 3]
```diff
@@ -2323,9 +2328,9 @@ Only the fields that you want to update are set.
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`storeCode`|string|optional|
-`entity`|string|required|
-`createDataKey`|string|required|
+`storeCode`|string|optional| ID of the store in which to apply the updated data.
+`entity`|string|required| The name of the updateData entity being created.
```
Comment (jilu1) [inline mftf/2.3/test/actions.md]: The data to be updated with

File: mftf/2.3/test/actions.md [hunk 4]
```diff
@@ -2027,9 +2032,9 @@ See [seeNumberOfElements docs on codeception.com](http://codeception.com/docs/mo
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`userInput`|string|optional|
-`parameterArray`|string|optional|
+`selector`|string|optional| The selector identifying the corresponding HTML element.
+`userInput`|string|optional| Number of instances of the specified selector to be found.
+`parameterArray`|string|optional| Array of parameters to be searched for within the selector.
```
Comment (jilu1) [inline mftf/2.3/test/actions.md]: select within the range of the result elements. e.g. [0, 10], select between 0 and 10 elements

File: mftf/2.3/test/actions.md [hunk 5]
```diff
@@ -1906,8 +1911,8 @@ See [seeInFormFields docs on codeception.com](http://codeception.com/docs/module
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`parameterArray`|string|optional|
+`selector`|string|optional| The selector identifying the corresponding HTML element.
+`parameterArray`|string|optional| Array of parameters to be searched for within the selector.
```
Comment (jilu1) [inline mftf/2.3/test/actions.md]: Array of form parameters (name => value) to be searched for within the selector

File: mftf/2.3/test/actions.md [hunk 6]
```diff
@@ -1846,8 +1851,8 @@ See [seeElementInDOM docs on codeception.com](http://codeception.com/docs/module
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`parameterArray`|string|optional|
+`selector`|string|optional| The selector identifying the corresponding HTML element.
+`parameterArray`|string|optional| Array of parameters to be searched for within the selected element.
```
Comment (jilu1) [inline mftf/2.3/test/actions.md]: Attributes of the elements

**Comment by jilu1** [comment]
> Li,
> There were 4 attributes I couldn't figure out. I would love suggestions for completing them.
> dontSeeElementInDOM
> parameterArray:
> attribute Array:

The schema needs to be cleaned up a bit. I think only one array type of attribute should be there. And it means "expected attributes of this element".

> formatMoney AND mSetLocale
> userInput:
> locale:
> I assume locale is a PHP locale value. Not sure what userInput would be…

userInput: the expected value amount in the given locale



> getData
> index:

index: the index in the returned data array.


> scrollTo
> selectorArray:

selectorArray: Selector in array format. like ['css' => '.checkout']

File: mftf/2.3/test/actions.md [hunk 7]
```diff
@@ -220,7 +220,7 @@ See [amOnPage docs on codeception.com](http://codeception.com/docs/modules/WebDr
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`url`|string|optional|
+`url`|string|optional| A path to the page relative to the MAGENTO_BASE_URL.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Done.

File: mftf/2.3/test/actions.md [hunk 8]
```diff
@@ -266,7 +266,7 @@ See [amOnUrl docs on codeception.com](http://codeception.com/docs/modules/WebDri
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`url`|string|optional|
+`url`|string|optional| Absolute URL to be used in subsequent steps.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Done.

File: mftf/2.3/test/actions.md [hunk 9]
```diff
@@ -408,10 +408,10 @@ See [clickWithLeftButton docs on codeception.com](http://codeception.com/docs/mo
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`selectorArray`|string|optional|
-`x`|string|optional|
-`y`|string|optional|
+`selector`|string|optional| The selector identifying the corresponding HTML element.
+`selectorArray`|string|optional| Selects an element as a key value array. See [strict locator]
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Done.

File: mftf/2.3/test/actions.md [hunk 10]
```diff
@@ -440,10 +440,10 @@ See [clickWithRightButton docs on codeception.com](http://codeception.com/docs/m
...
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Done.

File: mftf/2.3/test/actions.md [hunk 11]
```diff
@@ -1043,7 +1042,7 @@ See [executeJS docs on codeception.com](http://codeception.com/docs/modules/WebD
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`function`|string|optional|
+`function`|string|optional| Javascript to be executed.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Thanks.

File: mftf/2.3/test/actions.md [hunk 12]
```diff
@@ -1146,8 +1145,8 @@ See [grabAttributeFrom docs on codeception.com](http://codeception.com/docs/modu
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`userInput`|string|optional|
+`selector`|string|optional| The selector identifying the corresponding HTML element.
+`userInput`|string|optional| Name of tag attribute to grab.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: It's in the name of the function...

File: mftf/2.3/test/actions.md [hunk 13]
```diff
@@ -1510,8 +1514,8 @@ See [performOn docs on codeception.com](http://codeception.com/docs/modules/WebD
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`function`|string|optional|
+`selector`|string|optional| The selector identifying the corresponding HTML element.
+`function`|string|optional| Function or Actions to be taken on the selected element.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Thanks.

File: mftf/2.3/test/actions.md [hunk 14]
```diff
@@ -1790,7 +1794,7 @@ See [seeCurrentUrlEquals docs on codeception.com](http://codeception.com/docs/mo
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`url`|string|optional|
+`url`|string|optional| Full url to be searched for.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Done.

File: mftf/2.3/test/actions.md [hunk 15]
```diff
@@ -1948,7 +1952,7 @@ See [seeInPopup docs on codeception.com](http://codeception.com/docs/modules/Web
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`userInput`|string|optional|
+`userInput`|string|optional| String to be searched for within the popup
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Done.

File: mftf/2.3/test/actions.md [hunk 16]
```diff
@@ -2163,8 +2167,8 @@ See [switchToIFrame docs on codeception.com](http://codeception.com/docs/modules
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`userInput`|string|optional|
+`selector`|string|optional| The selector identifying the corresponding HTML element.
+`userInput`|string|optional| The name of the IFrame to set focus to.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Seems to be capital.

File: mftf/2.3/test/actions.md [hunk 17]
```diff
@@ -2183,7 +2187,7 @@ See [switchToNextTab docs on codeception.com](http://codeception.com/docs/module
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`userInput`|string|optional|
+`userInput`|string|optional| Offset of the tab to open, usually a number.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: I took that from the Codeception desc.

File: mftf/2.3/test/actions.md [hunk 18]
```diff
@@ -2207,7 +2211,7 @@ See [switchToPreviousTab docs on codeception.com](http://codeception.com/docs/mo
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`userInput`|string|optional|
+`userInput`|string|optional| Number of tabs to go back.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: In this case, 'tabs' are 'steps' like backing through a browser history.

File: mftf/2.3/test/actions.md [hunk 19]
```diff
@@ -2288,9 +2292,9 @@ See [unselectOption docs on codeception.com](http://codeception.com/docs/modules
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`userInput`|string|optional|
-`parameterArray`|string|optional|
+`selector`|string|optional| The selector identifying the corresponding HTML element.
+`userInput`|string|optional| The name of the option to deselect.
+`parameterArray`|string|optional| Array of options to be unselected.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Good catch.

File: mftf/2.3/test/actions.md (hunk 3 above)
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Nice.

File: mftf/2.3/test/actions.md [hunk 20]
```diff
@@ -2421,8 +2425,8 @@ See [waitForElementNotVisible docs on codeception.com](http://codeception.com/do
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`time`|string|optional|
+`selector`|string|optional| The selector identifying the corresponding HTML element.
+`time`|string|optional| The number of seconds to wait for the element to become non-visible.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Grabbed from Codeception. Changed to match the function name.

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
Closes #MAGEDOC-3114

**Pull request merged**